### ISSUE TITLE
[Bug] ListView is always missing first value when running Xamarin.UITests on iOS

### ISSUE THREAD

**Issue by wojcikjac** [xamarin/Xamarin.Forms#15354]
### Description

When I'm running UI Tests on iOS, first value on the ListView is always missing, and when I tap on the empty cell there is a crash. It looks like the DeviceAgent app that is running concurrently with the app and controls the UI Tests is somehow interfering with the ListView because as soon as I uninstall it the ListView is displayed correctly again.

It's happening on the newest Xamarin.Forms version (5.0.0.2401) but when I rolled back to 5.0.0.1874 the list was displayed correctly

![emptycell](https://user-images.githubusercontent.com/105010036/166982470-2945ad71-00b2-48f4-b3e7-0acd7e4bd64f.PNG)

**Comment by jfversluis** [comment]
Yeah something seems to have broken with the recycle strategy there. You seem like you are down in the Xamarin.Forms woods if you are checking our UITests? :) if you would be willing and able I'd appreciate your help on this one!

**Comment by wojcikjac** [comment]
What help do you need from me? It is me who is looking for help and for someone to fix this bug. I am not a Xamarin.Forms programmer/contributor, just a tester. And because of this bug I can't run UI Tests properly in the app I'm testing.

**Comment by roxton001** [comment]
Any updates on this guys?

**Comment by Sanoop-PM** [comment]
Any update on this issue? We are seeing this issue in Xamarin.Forms version (5.0.0.2612)

**Comment by anubPep122** [comment]
Was there any update on this issue, we are also facing this issue and now we are blocked
